### PR TITLE
Add golden json & fix submodule utils

### LINKED ISSUE #11: Change `Utils` directory from submodule to simple directory
Change `Utils` directory from a submodule to a simple directory

### LINKED ISSUE #13: Add golden json to post-proc.py
It should pick the correct golden json file info when running over data. Here, is an example:

https://github.com/ram1123/nanoAOD_skim/blob/ee7343a7170a1f3ead3476e732a35ee11949abad/post_proc.py#L23

### PR DESCRIPTION
**Pull request by ram1123**

### CODE REVIEW

File: PrintHist.py
Comment (ram1123) [inline PrintHist.py:1]: Add its documentation

File: Utils/check_condor_stuck_or_not.py
Comment (ram1123) [inline Utils/check_condor_stuck_or_not.py:1]: Add documentation

File: Utils/fileList.py
Comment (ram1123) [inline Utils/fileList.py:1]: documentation

File: scripts/GetLogSummary.py
Comment (ram1123) [inline scripts/GetLogSummary.py:1]: documentation?

**Review by ram1123 — COMMENTED** [review]
Add documentation for each python code.

**Comment by ram1123** [comment]
Hi @YujiLee301 ,

Please review this PR.  To avoid any mistakes, I changed the PR merge criteria. To merge a PR, it requires at least one other reviewer than the one who created the PR.

To get to know how to review the GitHub code please check this: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews

Please don't hesitate to let me know if you have any sissues.

**Review by YujiLee301 — APPROVED** [review]

**Review by YujiLee301 — APPROVED** [review]
Well aligned with previous cutflow

**Pull request merged**